imports/deps.json

```diff
@@ -0,0 +1,12 @@
+{
+  "meta": {
+    "fmt": {
+      "url": "https://denopkg.com/crewdevio/Trex@proxy/proxy/files/fmt.ts",
+      "hash": "5b4388adfdc1f5ad8a727f95a91c3f255cb36f430b6b4503f8f3ddacc162db53"
+    },
+    "flags": {
+      "url": "https://deno.land/std@0.66.0/flags/mod.ts",
+      "hash": "4f0d7fa435649c93d95f07eda25d45c77915b3643b8b97c094f66f098b95844e"
+    }
+  }
+}
```

Comment:
explaining a bit, this file is generated by trex imports, here it contains the url of the package and a hash generated based on the url of the package and the content of the package this for when a trex install is done check the integrity of both the url and of the content of the file this works as a snapshot, so if the content or the url is changed it will not be possible to download the package this is to guarantee the immutability of the package and prevent another version from being downloaded

Reply:
LGTM!